manifests/init.pp

```diff
@@ -369,6 +369,8 @@
     }
 
     class { 'pulpcore':
+      allowed_import_path       => $pulpcore_allowed_import_path,
+      allowed_export_path       => $pulpcore_allowed_export_path,
```

Comment:
These variables are undefined.

Reply:
You can refer to https://github.com/theforeman/puppet-foreman_proxy_content/commit/1d2eef1b71a5c1c5904f9447c05f7a3902cf5c1c for an example of what this should look like